awx/main/tests/unit/test_tasks.py

```diff
@@ -199,6 +200,7 @@ def setup_method(self, method):
         self.run_pexpect.return_value = ['successful', 0]
 
         self.patches = [
+            mock.patch.object(CallbackQueueDispatcher, 'dispatch', lambda obj: None),
```

Comment:
Don't actually dispatch callback events on the message bus in our tests - this makes the callback receiver listening on the other side of the bus very unhappy because the tests are sending nonsense :)